bash_completion.sh

```diff
@@ -456,8 +456,8 @@ _buildtest ()
       COMPREPLY=( $( compgen -W "${cmds}" -- $cur ) )
       ;;
     *)
-      local cmds="build buildspec cd cdash clean config debugreport docs help info inspect history path report schema schemadocs stats stylecheck tutorial-examples unittests"
-      local alias_cmds="bd bc cg debug it h hy rt style test"
+      local cmds="build buildspec cd cdash clean config debugreport docs help info inspect history path report schema schemadocs stats stylecheck tutorial-examples unittests commands"
```

Comment:
This list needs to be in alphabetical order so put `commands` in its right place